java/test/src/main/java/test/Ice/background/EndpointI.java

```diff
@@ -147,29 +149,18 @@ public EndpointI endpoint(com.zeroc.Ice.EndpointI delEndp) {
     }
 
     @Override
-    public java.util.List<com.zeroc.Ice.EndpointI> expandIfWildcard() {
-        java.util.List<com.zeroc.Ice.EndpointI> endps = new java.util.ArrayList<>();
-        for (com.zeroc.Ice.EndpointI endpt : _endpoint.expandIfWildcard()) {
-            endps.add(endpt == _endpoint ? this : new EndpointI(_configuration, endpt));
-        }
-        return endps;
+    public java.util.List<com.zeroc.Ice.EndpointI> expandHost() {
+        return _endpoint.expandHost().stream().map(this::endpoint).collect(Collectors.toList());
```

Comment:
This should work, and save some verbosity.
The only gotcha is that this returns an _immutable_ list (as does `Collections.singletonList`).
```suggestion
        return _endpoint.expandHost().stream().map(this::endpoint).toList();
```

Reply:
This doesn't work.